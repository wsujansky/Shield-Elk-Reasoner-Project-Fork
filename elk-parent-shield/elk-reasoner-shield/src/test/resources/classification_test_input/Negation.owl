Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Ontology(
SubClassOf(:A ObjectComplementOf(:B))
SubClassOf(:A :B)
)