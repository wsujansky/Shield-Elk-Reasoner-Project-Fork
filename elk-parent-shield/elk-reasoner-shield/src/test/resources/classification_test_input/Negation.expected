Ontology(
Declaration(Class(<http://example.org/A>))
Declaration(Class(<http://example.org/B>))
EquivalentClasses(<http://www.w3.org/2002/07/owl#Nothing> <http://example.org/A>)
)